trend_micro_vision_one_xdr/README.md

```diff
@@ -1,43 +1,65 @@
-# Agent Check: trend-micro-vision-one-xdr
-
 ## Overview
 
-This check monitors [trend-micro-vision-one-xdr][1].
+[Trend Micro Vision One XDR][1] collects and automatically correlates data across multiple security layers: email, endpoint, server, cloud workload, and network. This allows for faster detection of threats and improved investigation and response times through security analysis.
 
-## Setup
+This integration ingests the following logs:
 
-### Installation
+- Workbench Alerts: This endpoint contains information about all the standalone alerts triggered by detection models.
+- Observed Attack Techniques: This endpoint contains information about observed attack techniques from Detections, Endpoint Activity, Cloud Activity, Email Activity, Mobile Activity, Network Activity, Container Activity, and Identity Activity data sources.
 
-The trend-micro-vision-one-xdr check is included in the [Datadog Agent][2] package.
-No additional installation is needed on your server.
+This integration collects all the above listed logs and sends them to Datadog for analysis. Datadog uses the built-in logs pipeline to parse and enrich these logs, enabling effortless search and analysis. The integration provides insight into workbench alerts and observed attack techniques through the out-of-the-box dashboards. Also, This integration provides out of the box detection rules.
+
+## Setup
 
 ### Configuration
 
-!!! Add list of steps to set up this integration !!!
+#### Create API KEY from Trend Micro Vision One XDR
+
+1. On the Trend Vision One console, go to **Administration > API Keys** .
+2. Generate a new authentication token. Click **Add API key**. Specify the settings of the new API key.
```

Comment:
"Specify the settings of the new API key." > "Specify the settings of the new API key with the following:"

Reply:
Done